### ISSUE TITLE
Assassin's Creed Odyssey blotches on top of lava

### ISSUE THREAD

**Issue by AccountOneOff** [doitsujin/dxvk#944]
The lava has some black spots on top of it that flicker.

![screenshot from 2019-02-28 21-01-45](https://user-images.githubusercontent.com/15083452/53622658-661cf480-3baf-11e9-885e-2a10eb97540e.png)

<details>
<summary>Screenshots of location</summary>

![screenshot from 2019-02-28 23-30-41](https://user-images.githubusercontent.com/15083452/53623130-0b849800-3bb1-11e9-9a86-fc482dfe1e27.png)

![screenshot from 2019-02-28 22-49-37](https://user-images.githubusercontent.com/15083452/53622664-6ae1a880-3baf-11e9-8686-3987dcd9134c.png)
</details>

### Software information
Medium preset

### System information
- GPU: GTX 960
- Driver: 415.27
- Wine version: Staging 4.2
- DXVK version: 1.0

### Apitrace file(s)
ACO crashes or hangs (had both happen) when interacting with the main menu. I know it won't help debug the issue at hand but maybe it'll help understand why it doesn't get through the main menu.

https://drive.google.com/open?id=1vqZLUyWLia5YhYnfNeG7kYK4gUnRw3eh

### Log files
- [ACOdyssey_d3d11.log](https://github.com/doitsujin/dxvk/files/2918301/ACOdyssey_d3d11.log)
- [ACOdyssey_dxgi.log](https://github.com/doitsujin/dxvk/files/2918302/ACOdyssey_dxgi.log)

**Comment by ah-** [comment]
Looks somewhat similar to earlier AC:O bugreports, could you try again with 418.31.03?

**Comment by doitsujin** [comment]
Works fine for me:
![bildschirmfoto-607](https://user-images.githubusercontent.com/25567304/53628579-b1fd8700-3c0b-11e9-914c-ea8de0172a24.png)

Please test the latest Nvidia beta driver as suggested above.

**Comment by doitsujin** [comment]
Closing due to lack of feedback, issue should be fixed as of Nvidia 418.43.

**Comment by AccountOneOff** [comment]
Yeah sorry about that been a bit busy. Same shit happens with 418.43

I've tried low settings and disabling nvapi hack but nothing same result.

![Screenshot from 2019-03-14 18-31-06](https://user-images.githubusercontent.com/15083452/54402658-03852780-468a-11e9-97e7-798e4d6ec2b9.png)

**Comment by doitsujin** [comment]
Can you test the latest Vulkan beta driver?

**Comment by AccountOneOff** [comment]
Like I mentioned on discord the last time that depends on Ubuntu updating their drivers. Unfortunately it looks like they're back to only supporting stable releases. :unamused:

**Comment by AccountOneOff** [comment]
Still happening with 418.56

![Screenshot from 2019-03-25 21-54-07](https://user-images.githubusercontent.com/15083452/54972532-33b1ad80-4f49-11e9-8c2b-e6022818824c.png)

[ACOdyssey_d3d11.log](https://github.com/doitsujin/dxvk/files/3006551/ACOdyssey_d3d11.log)
[ACOdyssey_dxgi.log](https://github.com/doitsujin/dxvk/files/3006552/ACOdyssey_dxgi.log)

**Comment by doitsujin** [comment]
Can you re-test with the latest Vulkan beta driver?

**Comment by doitsujin** [comment]
Closing due to lack of feedback.

**Comment by Vash63** [comment]
I finally found a location where I can reproduce this. Tested on 440.59 and 440.48.02.

[odysseysaves.tar.gz](https://github.com/doitsujin/dxvk/files/4150743/odysseysaves.tar.gz)

It's the 'manual save' about the 4th one down from the load game link. The way the saves are listed in game doesn't match the files at all so I'm not sure how to send only the single one (and the timestamps don't line up). I'll try to get an apitrace later.
